daisy/step_create_images.go

```diff
@@ -54,11 +54,15 @@ func (ci *CreateImages) UnmarshalJSON(b []byte) error {
 
 func imageUsesAlphaFeatures(imagesAlpha []*ImageAlpha) bool {
 	for _, imageAlpha := range imagesAlpha {
-		if imageAlpha != nil && len(imageAlpha.RolloutOverride.DefaultRolloutTime) > 0 {
-			return true
+		if imageAlpha != nil && imageAlpha.RolloutOverride != nil {
+			if len(imageAlpha.RolloutOverride.DefaultRolloutTime) > 0 {
```

Comment:
minor, you can squash these into one `if`

Reply:
Updated in [99c3847](https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1608/commits/99c3847ba83eaf50d1fc32d551bd5343d185b45e)